### DISCUSSION TITLE
Mechanism for SPARQL extensions?

### DISCUSSION THREAD

**Discussion by rybesh** [oxigraph/oxigraph#919] [Q&A] [answered]
Are there any plans to support SPARQL extensions such as [GeoSPARQL](https://www.ogc.org/standard/geosparql/)?

(Note that my question is not about whether oxigraph plans to implement GeoSPARQL support, but about whether there is a mechanism that would allow others to write plugins that extend the core SPARQL support.)

**Comment by Tpt** [comment, marked as answer]
Yes! Oxigraph supports custom functions so implementing GeoSPARQL should be possible (see #172)
